src/esp/io/io.h

```diff
@@ -19,6 +20,14 @@ std::string removeExtension(const std::string& file);
 
 std::string changeExtension(const std::string& file, const std::string& ext);
 
+/**
+ * @brief This function will perform globbing on a passed path + pattern,
+ * returning all the files and directories that match the pattern.
+ * @param pattern The pattern to match
```

Comment:
Maybe mention what is allowed in the pattern? As in, it's just wildcards and not full regular expressions.

Reply:
Is there a way in the docs to put an external html link? (I was thinking of linking wiki :D )

Reply:
It's a standard Markdown, so `[title](url)` works. Or just put an URL there directly.